test/integration2/query/misc/additional.spec.js

```diff
@@ -258,26 +258,63 @@ describe('Additional', function () {
             return this.skip();
           }
 
-          const res = await knex('accounts_foo')
+          const insertRes = await knex('accounts_foo')
             .insert({ balance_foo: 123 })
             .returning('balance_foo');
-          expect(res).to.eql([
+          expect(insertRes).to.eql([
             {
               balance_foo: 123,
             },
           ]);
+
+          const updateRes = await knex('accounts_foo')
+            .where({ balance_foo: 123 })
+            .update({ balance_foo: 456 })
+            .returning('balance_foo');
+          expect(updateRes).to.eql([
+            {
+              balance_foo: 456,
+            },
+          ]);
+
+          const delRes = await knex('accounts_foo')
+            .where({ balance_foo: 456 })
+            .del()
+            .returning('balance_foo');
+          expect(delRes).to.eql([
+            {
+              balance_foo: 456,
+            },
+          ]);
         });
 
         it('should return the correct columns when multiple properties are given to returning', async function () {
           if (!isPostgreSQL(knex) && !isMssql(knex) && !isBetterSQLite3(knex)) {
```

Comment:
does this still not work with node-sqlite? if it does, can you check all usages for `isBetterSQLite3` and see if we can test node-sqlite there too?

Reply:
No, `returning` still not work with `node-sqlite3`.
Only `better-sqlite3` pass the test.
https://github.com/knex/knex/compare/master...yodhcn:knex:test_node-sqlite3_returning_support